src/components/common/Select.js

```diff
@@ -130,3 +132,65 @@ export const AsyncCreatableSelect = props => {
     ...props
   })
 }
+
+const VirtualizedMenuList = props => {
+  const { children, focusedOption, getStyles, getValue, maxHeight } = props
+
+  const list = useRef()
+
+  // Scroll to the currently selected value (if there is one) when the menu is opened.
+  useOnMount(() => {
+    const [value] = getValue()
+    if (value) {
+      const renderedOptions = Children.map(children, child => child.props.data)
+      const valueIndex = renderedOptions.findIndex(opt => opt.value === value.value)
+      if (valueIndex !== -1) {
+        list.current.scrollToRow(valueIndex)
+      }
+    }
+  })
+
+  // When navigating option with arrow keys, scroll the virtualized list so that the focused option is visible.
+  useEffect(() => {
+    const renderedOptions = Children.map(children, child => child.props.data)
```

Comment:
Nit: could make a helper method out of lines 155-159 and use it for lines 145-149 also.

Reply:
Done in 93925ed4f5e7da406b7bd10f952812b297f8d6f6.